tests/Makefile

```diff
@@ -1,132 +1,252 @@
+        --install build/orbit-simulator/charts/orbit-simulator \
+        --set simulator.shim.cloudserver_release=$(ZENKO_HELM_RELEASE) \
+        --wait $(shell ../eve/workers/ci_env.sh set)
+endif
+.PHONY: install-orbit-simulator
+
+install-cosbench:
+	git clone git@github.com:scality/zenko-cosbench.git build/zenko-cosbench && \
+	$(HELM) upgrade $(COSBENCH_HELM_RELEASE) \
+		--namespace $(HELM_NAMESPACE) \
+		--install build/zenko-cosbench/charts/zenko-cosbench \
+		--wait
+.PHONY: install-cosbench
+
+install-zenko:
+ifndef NO_INSTALL
+	$(HELM) dependency build ../charts/zenko && \
+	$(HELM) upgrade $(ZENKO_HELM_RELEASE) \
+		--namespace $(HELM_NAMESPACE) \
+		--install ../charts/zenko/ \
+		-f ../eve/ci-values.yml
+endif
+.PHONY: install-zenko
+
+install-latest-zenko:
+ifndef NO_INSTALL
+	$(HELM) dependency build ../charts/zenko && \
+	$(HELM) upgrade $(ZENKO_HELM_RELEASE) \
+		--namespace $(HELM_NAMESPACE) \
+		--install ../charts/zenko/ \
+		-f ../eve/ci-values.yml \
+        --set cloudserver.image.repository=$(IMAGE_REGISTRY)/zenko/cloudserver \
+        --set-string cloudserver.image.tag='$(CLOUDSERVER_TAG)' \
+        --set backbeat.image.repository=$(IMAGE_REGISTRY)/zenko/backbeat \
+        --set-string backbeat.image.tag='$(BACKBEAT_TAG)' \
+        --set s3-data.image.repository=$(IMAGE_REGISTRY)/zenko/cloudserver \
+        --set-string s3-data.image.tag='$(CLOUDSERVER_TAG)'
+endif
+.PHONY: install-latest-zenko
+
+# Target to install common components need for all tests
+install-common: | install-tiller install-helm-repos install-orbit-simulator
+.PHONY: install-common
+
+wait-for-zenko: | build-e2e
+	$(KUBECTL) run zenko-waiter \
+		--image $(E2E_IMAGE) \
+		--rm \
+		--attach=True \
+		--restart=Never \
+		--namespace=$(HELM_NAMESPACE) \
+		$(shell ../eve/workers/ci_env.sh env) \
+		--command -- python create_buckets.py || (echo "Zenko has failed to stabilize" ; kubectl get pods -L redis-role ; exit 1) >&2
+.PHONY: wait-for-zenko
+
+dump-logs: | artifacts
+	@(kubectl  -n $(HELM_NAMESPACE) get pods -o go-template --template '$(ZENKO_PODS)') | xargs -n 1 -r sh -c;  \
+	(kubectl -n $(HELM_NAMESPACE) get pods -o jsonpath='$(E2E_LOG_PODS)') | \
 	while read -r pod; do \
-		kubectl -n $(ZENKO_K8S_NAMESPACE) logs $${pod}; \
+		kubectl -n $(HELM_NAMESPACE) logs $${pod}; \
 	done;
 .PHONY: dump-logs
 
-cleanup:
-	@rm -rf artifacts/*
-	@($(KUBECTL) get pod -o=custom-columns=NAME:.metadata.name --no-headers=true) | \
-	grep $(E2E_POD) | xargs -r kubectl delete pod
-
-run-all: | helm-test end2end dump-logs cleanup
-
-local-run: | cleanup zenko-run end2end dump-logs
-
-lint:
-	@$(TOX)
-.PHONY: lint
+test-e2e: | build-e2e
+	$(KUBECTL) run $(E2E_POD) \
+		--rm \
+		--restart=Never \
+		--attach=True \
+		--image=$(E2E_IMAGE)  \
+		--namespace $(HELM_NAMESPACE) \
+		$(shell ../eve/workers/ci_env.sh env) || (echo "Tests have failed" ; kubectl get pods -L redis-role ; exit 1) >&2
+.PHONY: test-e2e
+
+test-backbeat: | build-bb-e2e
```

Comment:
We should change this to something like `test-node` or `run-node-tests`. I know these tests were changed to up to be more generic. Because like `test_location_quota` is an Cloudserver test.